src/Umbraco.Web.UI.Client/src/views/propertyeditors/templatablegrid/templatablegrid.html

```diff
@@ -0,0 +1,3 @@
+﻿<div ng-controller="Umbraco.PropertyEditors.TemplatableGridController">
+    <div ng-include="'/umbraco/views/propertyeditors/grid/grid.html'"></div>
```

Comment:
Sorry, not quite sure how to find respective paths. Relative or just name doesn't seem to work.